mev_inspect/block.py

```diff
@@ -51,7 +51,7 @@ def fetch_block(w3, base_provider, block_number: int) -> Block:
 
     ## Get gas used by individual txs and store them too
     txs_gas_data: Dict[str, Dict[str, Any]] = {}
-
+    """
```

Comment:
meant to be here still?

we should probably delete this code for now if it's not being used since it causes block loading to be slow

Reply:
Wasn't clear on if it were to be added back in.  Removed for the time being.